pip/pep425tags.py

```diff
@@ -124,7 +124,58 @@ def get_platform():
         split_ver = release.split('.')
         return 'macosx_{0}_{1}_{2}'.format(split_ver[0], split_ver[1], machine)
     # XXX remove distutils dependency
-    return distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    result = distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    if result == "linux_x86_64" and sys.maxsize == 2147483647:
+        result = "linux_i686"
+    return result
+
+
+def is_manylinux1_compatible():
+    # Only Linux, and only x86-64 / i686
+    if get_platform() not in ("linux_x86_64", "linux_i686"):
+        return False
+
+    # "wide" Unicode mode is mandatory (always true on CPython 3.3+)
+    if sys.maxunicode <= 0xFFFF:
+        return False
```

Comment:
I think the only change needed to handle this last round of PEP updates is to delete this unicode check